examples/ec2_remote/package.json

```diff
@@ -6,6 +6,7 @@
     },
     "dependencies": {
         "@pulumi/aws": "^4.18.0",
+        "@pulumi/command": "^0.5.2",
```

Comment:
I think this should point to `"../../sdk/nodejs"` (or whatever the correct syntax is).